src/components/SingleReview.jsx

```diff
@@ -2,11 +2,14 @@ import { useEffect, useState } from "react"
 import { getReviewById } from "../api";
 import { useParams } from "react-router-dom"
 import { Comments } from "./Comments";
+import { Vote } from "./Vote";
 
 export const SingleReview = () => {
     const [singleReview, setSingleReview] = useState({});
     const {review_id} = useParams();
     const [isLoading, setIsLoading] = useState(true);
+    const [incVotes, setIncVotes] = useState(0);
```

Comment:
These currently do nothing as your optimistic rendering is not working. I would have a look at what you could hold in state to render on the page that you COULD optimistically change.